cmd/runtimecfg/node-ip.go

```diff
@@ -252,7 +259,13 @@ func getSuitableIPs(retry bool, vips []net.IP, preferIPv6 bool, networkType stri
 		}
 
 		log.Errorf("Failed to find a suitable node IP")
-		time.Sleep(time.Second)
+		if timerLoop > maxSecondsToSuitableIPsLoop {
```

Comment:
Probably not a realistic problem, but I would cap timerLoop to maxSeconds... instead of changing the Sleep call. No chance of timerLoop overflowing its type that way.

Reply:
Please see if the next patch address the request.